### DISCUSSION TITLE
最新版的fastgpt在docker下mongo无法启动，最近的更新mongo都起不来

### DISCUSSION THREAD

**Discussion by DeanQi** [labring/FastGPT#917] [Q&A]
最新版的fastgpt在docker下mongo无法启动，最近的更新mongo都起不来，请问如何解决？

**Comment by c121914yu** [comment]
> 最新版的fastgpt在docker下mongo无法启动，最近的更新mongo都起不来，请问如何解决？

https://github.com/labring/FastGPT/issues/823 重复

**Comment by 13874360430** [comment]
fastgpt在docker下mongo无法启动，mongo起不来，请问如何解决？
